cli/src/test/java/com/devonfw/tools/ide/environment/EnvironmentVariablesPropertiesFileTest.java

```diff
@@ -6,40 +6,53 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
 
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
 import com.devonfw.tools.ide.context.IdeTestContextMock;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+import com.devonfw.tools.ide.version.VersionIdentifier;
 
 /**
  * Test of {@link EnvironmentVariablesPropertiesFile}.
  */
 @SuppressWarnings("javadoc")
-class EnvironmentVariablesPropertiesFileTest extends Assertions {
+class EnvironmentVariablesPropertiesFileTest extends AbstractIdeContextTest {
 
   /**
    * Test of {@link EnvironmentVariablesPropertiesFile} including legacy support.
    */
 
```

Comment:
What happened here? Shouldn't this be moved?

Reply:
This was cause by some git merge since that is not merging semantically. I moved the JavaDoc back to the method where it was before.